src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -359,7 +359,7 @@ public static bool TryWriteBytes(Span<byte> destination, float value)
         public static byte[] GetBytes(double value)
         {
             byte[] bytes = new byte[sizeof(double)];
-            Unsafe.As<byte, double>(ref bytes[0]) = value;
```

Comment:
This was an actual bug since arrays only get ptr alignment so this could break on arm32.

Reply:
@xtqqczze I don't think you should wait till it's fixed, these APIs allocate, a tiny, almost free `lea` won't make a difference, I think you should use Write there